src/openrct2/peep/RideUseSystem.cpp

```diff
@@ -0,0 +1,26 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2021 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#include "RideUseSystem.h"
+
+namespace OpenRCT2::RideUse
+{
+    static RideHistory _history;
+    static RideTypeHistory _typeHistory;
```

Comment:
How do we go about avoiding new globals? Can this be part of a game state?

Reply:
yes game state can contain this.

Reply:
I think we should let this slip for now. Do also properly address this can also better organize this I think, we should create a new Milestone addressing globals and identify code and create corresponding issues. The GetContext() thing on its own is exactly the anti pattern we try to avoid really.